README.md

```diff
@@ -142,20 +142,21 @@ optional arguments:
 In the folder where you are running Commander, create a file called config.json which defines the
 API server and the email address.
 
-For customers hosted in the US region, set the US endpoint:
-```shell script
-{
-  "server": "https://keepersecurity.com/api/v2/"
-}
-```
+Commander defaults to the US region, so customers hosted in other regions will need to specify a server in the config.
 
-For customers hosted in the EU region, set the EU endpoint:
+####Example config.json for EU region
 ```shell script
 {
-  "server": "https://keepersecurity.eu/api/v2/"
+  "server": "https://keepersecurity.eu"
 }
 ```
 
+| Region  | URL                           | Property in `config.json` file              |
+|---------|-------------------------------|---------------------------------------------|
+| US      | https://keepersecurity.com    | ```{"server": "https://keepersecurity.com"}```    |
```

Comment:
How about also specifying that for us it is optional to have this in the config

Reply:
I did mention it was the default, and that only users of other regions need to set it. I don't think it's useful information to further say it's optional here

Reply:
Ok, that's fine